app/client/api/creditTransferAPI.js

```diff
@@ -1,13 +1,13 @@
 import { getToken, handleResponse, generateQueryString } from '../utils'
 
 export const loadCreditTransferListAPI = ({query , path}) => {
+    var query_string = generateQueryString(query);
     if (query) {
-        const query_string = generateQueryString(query);
         var URL = `/api/credit_transfer/${query_string}`;
     } else if (path) {
-        URL = `/api/credit_transfer/${path}/`;
+        URL = `/api/credit_transfer/${path}/${query_string}`;
     } else {
-        URL = '/api/credit_transfer/';
+        URL = `/api/credit_transfer/${query_string}`;
```

Comment:
I used to be put off by string literals, now I lovvvve them 😂

Reply:
I feel like we could have a `generateFormattedURL(`/api/credit_transfer/`, query, path)` util that generates a correct URL automatically ...

Reply:
I was literally gonna suggest the same thing, but I couldn’t find any other location where we need to parse the path.

Reply:
cool refactored